Resources/views/notifications.html.twig

```diff
@@ -1,35 +1,55 @@
 {% for flashMessage in app.session.flashbag.get('success') %}
```

Comment:
Je hebt nu 4 keer bijna dezelfde code in 1 bestand, dat lijkt mij nu niet echt ideaal. Een macro lijkt mij wel handig hiervoor.